05_iigo/index.tex

```diff
@@ -347,50 +348,67 @@ \section{Accountability Cycle}
 
 \subsection{Transfer-of-power}
 \label{subsec:transfer-of-power}
+
+For the scope of this project we chose only one system of power transfer for the islands to utilise, and it is an election. The islands that hold institutional power are the decision group of the archipellago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles, who makes the aforementioned decisions. An election also opens up another avenue for opinion formation to have an effect. 
+\begin{itemize}
+    \item Each role has the \emph{power} to call an election vote and declare the winner (see Figure~ \ref{fig:cycles_in_IIGO} for the power transfer cycle). 
+\end{itemize}
+
+We note that:
 \begin{enumerate}
-    \item The Speaker conducts a vote for the election of the new Judge.
-    \item The Judge conducts a vote for the election of the new President.
-    \item The President conducts a vote for the election of the new Speaker.
+    \item The Speaker conducts an election for the election of the new Judge.
+    \item The Judge conducts an election for the election of the new President.
+    \item The President conducts an election for the election of the new Speaker.
```

Comment:
can we not use election twice in one sentence here? i've suggested appoint but whatever rewording you would prefer is good
```suggestion
    \item The Speaker conducts an election to appoint a new Judge.
    \item The Judge conducts an election to appoint a new President.
    \item The President conducts an election to appoint a new Speaker.
```

Reply:
ahahahaha wtf